build-tools/enumification-helpers/README

```diff
@@ -0,0 +1,256 @@
+
+This directory contains a couple of "helper" tools for API int-to-enum conversion.
```

Comment:
Regarding "making changes before porting", we have two options: keep consistent with monodroid/oss-history or do not import